src/ui/pages/Overview/BackupReminder.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useCallback, useEffect, useRef } from 'react';
+import { BottomSheetDialog } from 'src/ui/ui-kit/ModalDialogs/BottomSheetDialog';
+import type { HTMLDialogElementInterface } from 'src/ui/ui-kit/ModalDialogs/HTMLDialogElementInterface';
+import { VStack } from 'src/ui/ui-kit/VStack';
+import { WarningIcon } from 'src/ui/components/WarningIcon';
+import { UIText } from 'src/ui/ui-kit/UIText';
+import { HStack } from 'src/ui/ui-kit/HStack';
+import { Button } from 'src/ui/ui-kit/Button';
+import { usePreferences } from 'src/ui/features/preferences';
+import { UnstyledLink } from 'src/ui/ui-kit/UnstyledLink';
+import { useBackupTodosCount } from '../BackupWallet/useBackupTodosCount';
+
+// 2 mins break for testing purpose
+const ONE_DAY = 1000 * 60 * 2; // 1000 * 60 * 60 * 24;
+
+export function BackupReminder() {
```

Comment:
Let's maybe create a wrapper component where we determine if dialog needs to be rendered at all so that we're not mounting any DOM elements when this dialog isn't needed?